azurecompute/src/main/java/org/jclouds/azurecompute/options/AzureComputeTemplateOptions.java

```diff
@@ -89,6 +89,14 @@ public void copyTo(final TemplateOptions to) {
          if (storageAccountType.isPresent()) {
             eTo.storageAccountType(storageAccountType.get());
          }
+
+         final LoginCredentials credentials = new LoginCredentials.Builder().
+                 user(loginUser).
+                 password(loginPassword == null || !loginPassword.isPresent() ? null : loginPassword.get()).
+                 privateKey(loginPrivateKey == null || !loginPrivateKey.isPresent() ? null : loginPrivateKey.get()).
+                 authenticateSudo(authenticateSudo == null ? false : authenticateSudo).
+                 build();
+         eTo.overrideLoginCredentials(credentials);
```

Comment:
>WDYT? Shall I open an issue / PR about this and leave the fix temporarily here (in order to keep the test case above working)?

If you can do that, that would be much appreciated. Let's fix jclouds as soon as we find some issues instead of silo-ing fixes in providers.